tests/modules/test_structureset.py

```
@@ -1,24 +1,17 @@
-import pytest
-import numpy as np
-from unittest.mock import MagicMock, patch
+import pathlib
```

Ruff (0.8.2)

`pathlib` imported but unused

Remove unused import: `pathlib`

(F401)